data/casestudies.json

```diff
@@ -0,0 +1,59 @@
+[
```

Comment:
Can you please build a real data set with our case studies? This is the list:
https://json-schema.org/blog?type=Case%20Study

Reply:
Hey @benjagm 
where do we want to direct the link we are adding in each casestudy card? According to card's implementation the link I am adding is shown as read more.
e.g if I have added homepage url in link prop of card it is showing read more and on clicking read more i am directed to home page.

Reply:
> where do we want to direct the link we are adding in each casestudy card?

The url will be the case study blog post.

Reply:
Okay! Got it

Reply:
Hey @benjagm I have some doubts regarding data set
What are we going to add at the place of logo and are we going to add complete title of blog post if yes then it will be too large because in the implementation of card title it is made by using h3 so how should I proceed with it.

Reply:
We need to find the company logos and add them to the folder: https://github.com/json-schema-org/website/tree/main/public/img/logos

Reply:
Why changing the size of the card? Try to use this one and if you need to make some adjustments do it but just for the card features that you need. Remember that the card is now going to be used in 3 different pages. 

For example you can change the logo size or fix a bug but not the whole card component without reason. Right?

Reply:
Sorry for the confusion @benjagm 
I am not asking to change the card size but the bug is related to title if I directly put title in this component it will look like this

![cr](https://github.com/json-schema-org/website/assets/96949249/eb708746-bf29-4eff-a040-1147a8b6dbe9)

It is  because title is written with h3 size in card component.
I will definitely not change the card component directly but if there is no other way I will just make a copy of this component and make changes to that one .

Reply:
In this case feel free to modify it